.github/workflows/cargo-update.yml

```diff
@@ -21,12 +17,26 @@ jobs:
       - name: Cargo update
         run: cargo update
 
+      - name: Commit changes
+        id: commit
+        continue-on-error: true
+        run: |
+          git config user.name 'Phylum Bot'
+          git config user.email 'phylum-bot@users.noreply.github.com'
```

Comment:
<img width="1006" alt="Screen Shot 2022-06-13 at 9 14 57 AM" src="https://user-images.githubusercontent.com/18729796/173374337-0078d6ac-cb98-4da7-9861-6d3d3a4ffcdb.png">

To get the verified commits for `phylum-bot`, the `phylum-dev/phylum-ci` repository had to:
* Set the user name/email to `phylum-bot <69485888+phylum-bot@users.noreply.github.com>`
* Import the matching GPG key for signing commits:

```yml
      - name: Import GPG key for bot account
        uses: crazy-max/ghaction-import-gpg@v4
        with:
          gpg_private_key: ${{ secrets.PHYLUM_BOT_GPG_PRIVATE_KEY }}
          passphrase: ${{ secrets.PHYLUM_BOT_GPG_PASSPHRASE }}
          git_user_signingkey: true
          git_commit_gpgsign: true
          git_tag_gpgsign: true
```
It may not be absolutely necessary to sign these commits. However, if the signing option is pursued, it is worth also considering how to properly protect the secrets. For instance, in the `phylum-ci` repo, the bot related secrets are protected with the `Production` environment...which has limited access.

Reply:
It's not at all necessary to sign these commits. They can and should be signed when the merge occurs. And just like when this was first discussed ([here](https://github.com/phylum-dev/cli/pull/339#issuecomment-1116305063)), we still do not enforce a signature requirement on any branch.

On a philosophical level, this can be treated just like a PR coming in from a forked repository. There is no need to assign any trust to @phylum-bot because an authorized human will review and merge the PR.